bookkeeper-common/src/main/java/org/apache/bookkeeper/common/collections/RecyclableArrayList.java

```diff
@@ -64,4 +64,15 @@ public void recycle() {
             handle.recycle(this);
         }
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
why do we need these? it seems to call `super.equals`?

Reply:
Spot bugs is not happy